File: TestesPython/paga_meia.py
idade = int(input("Qual a sua idade ? "))
if idade <= 21 :
  print("Você paga meia entrada no cinema pela idade !")
elif idade > 21 :
    escpub = str(input("Você trabalha em Escola pública ? [Sim/Não] "))
    if escpub == "Sim" :
      print("Você não paga meia entrada no cinema pela idade, porém por trabalhar em Escola pública terá direito a meia entrada no cinema !")  
    elif escpub == "Não" :
      print("Você não terá direito a meia entrada no cinema pois já tem mais de 21 anos e não trabalha em Escola pública !")  
    

File: TestesJS/paga_meia.js
idade = 31
escpub = "Não" 
if (idade < 21) {
    console.log("Você ainda paga meia no cinema pela idade !")
}
else if ( idade >= 21) {
    if (escpub == "Sim") {
        console.log("Você não paga meia no cinema pela idade, porém por trabalhar em Escola Pública terá direito a meia entrada")
        
    }
    else if ( escpub == "Não") {
        console.log("Você não paga meia no cinema e por ter mais de 21 Anos e por não trabalhar em Escola Pública !")
    }

   
}

